src/argowrapper/engine/helpers/argo_engine_helper.py

```diff
@@ -216,7 +216,7 @@ def convert_string_to_pod_label(value: str) -> str:
     return label
 
 
-def get_username_from_token(header_and_or_token: str) -> str:
+def get_username_from_token(header_and_or_token: Union[str, None]) -> str:
```

Comment:
why not just

```
def get_username_from_token(header_and_or_token: Optional[str]) -> str:
```
or just how it was (`header_and_or_token: str`)...since I don't think it is really optional.

Reply:
Fixed. 👍